phing/tasks/setup.xml

```diff
@@ -289,4 +289,17 @@
     </if>
   </target>
 
+  <!-- Create a subtheme and set it as the site default -->
+  <target name="setup:subtheme" description="Creates a new subtheme of Cog.">
+    <drush command="en cog -y" verbose="false" logoutput="true" />
+    <drush command="cog ${project.subtheme.machine_name} ${project.subtheme.human_name} --description=${project.subtheme.description}" verbose="false" dir="${docroot}/themes/contrib/cog" logoutput="true" />
```

Comment:
It looks like this will run every single time that "setup" is executed. We should probably make this generation a bit more intentional and couple it less tightly to BLT's core targets.

For example, create a `frontend:cog:init` command that will:
* Ensure cog is in composer.json
* Enable cog
* Generate a subtheme
* Enable subtheme
* Set config

This way it can be run once and then not again.